### ISSUE TITLE
Segfaults

### ISSUE THREAD

**Issue by peterbraden** [hoprnet/hopr-connect#35]
For example:

```
2021-01-12T13:13:23.113Z hopr-connect:listener:error inbound connection failed Error: Value is null
    at Object.readLP (/Users/peterbraden/test/node_modules/it-pb-rpc/src/index.js:37:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async exchange (/Users/peterbraden/test/node_modules/libp2p-secio/src/handshake/exchange.js:18:15)
    at async handshake (/Users/peterbraden/test/node_modules/libp2p-secio/src/handshake/index.js:11:3)
    at async Object.secure (/Users/peterbraden/test/node_modules/libp2p-secio/src/index.js:22:3)
    at async Upgrader._encryptInbound (/Users/peterbraden/test/node_modules/libp2p/src/upgrader.js:358:12)
    at async Upgrader.upgradeInbound (/Users/peterbraden/test/node_modules/libp2p/src/upgrader.js:96:11)
    at async Listener.onTCPConnection (/Users/peterbraden/test/node_modules/@hoprnet/hopr-connect/lib/listener.js:179:20) {
  code: 'ERR_ENCRYPTION_FAILED'
}
2021-01-12T13:13:23.113Z hopr-connect:listener:error inbound connection failed Error: Value is null
    at Object.readLP (/Users/peterbraden/test/node_modules/it-pb-rpc/src/index.js:37:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async exchange (/Users/peterbraden/test/node_modules/libp2p-secio/src/handshake/exchange.js:18:15)
    at async handshake (/Users/peterbraden/test/node_modules/libp2p-secio/src/handshake/index.js:11:3)
    at async Object.secure (/Users/peterbraden/test/node_modules/libp2p-secio/src/index.js:22:3)
    at async Upgrader._encryptInbound (/Users/peterbraden/test/node_modules/libp2p/src/upgrader.js:358:12)
    at async Upgrader.upgradeInbound (/Users/peterbraden/test/node_modules/libp2p/src/upgrader.js:96:11)
    at async Listener.onTCPConnection (/Users/peterbraden/test/node_modules/@hoprnet/hopr-connect/lib/listener.js:179:20) {
  code: 'ERR_ENCRYPTION_FAILED'
}
zsh: segmentation fault  DEBUG=hopr* ./node_modules/.bin/hoprd --admin --init 2>&1 | 
zsh: done                tee logs.txt
```

**Comment by robertkiel** [comment]
most likely fixed with 0.2.7

**Comment by robertkiel** [comment]
No longer occuring since 0.2.7
